src/hotspot/cpu/aarch64/interpreterRT_aarch64.cpp

```diff
@@ -320,7 +414,7 @@ class SlowSignatureHandler
       *_int_args++ = (*from_addr == 0) ? NULL : (intptr_t)from_addr;
       _num_int_args++;
     } else {
-      *_to++ = (*from_addr == 0) ? NULL : (intptr_t) from_addr;
+      store_and_inc(_to, (*from_addr == 0) ? (intptr_t)NULL : (intptr_t) from_addr, wordSize);
       _num_int_args++;
```

Comment:
`pass_byte()` and  `pass_short()` use only one `_num_int_args++;` after the `if else` but other methods use it 2 of them inside `if else`

We should be consistent.